samples/bluetooth/bap_broadcast_assistant/src/main.c

```diff
@@ -480,7 +481,8 @@ static void disconnected(struct bt_conn *conn, uint8_t reason)
 
 	printk("Disconnected: %s, reason 0x%02x %s\n", addr, reason, bt_hci_err_to_str(reason));
 
-	bt_conn_unref(broadcast_sink_conn);
+	bt_unpair(BT_ID_DEFAULT, NULL);
```

Comment:
I don't think it makes sense to support bonding and then unpair on disconnect. 

Either we support bonding, or we don't :) (and it's required to support for LE Audio)

If you are experiencing issues with the current bonding support, we should look into alternatives such as overwriting bonds when there's a new bond only.

Reply:
Can we discuss?